core/src/main/kotlin/io/rover/campaigns/core/data/domain/Location.kt

```diff
@@ -42,7 +42,7 @@ data class Location(
             longitude = location.coordinate.longitude
         }
 
-        return loc1.distanceTo(loc2) < 100
+        return loc1.distanceTo(loc2) < 500f
```

Comment:
how about pulling this number out into a constant, and renaming this function to `isNotSignificantDisplacement` or something?